### ISSUE TITLE
Change jQuery version

### ISSUE THREAD

**Issue by videogreg93** [deepnight/ldtk#187]
Was trying to set the exact value of percentage on an auto tile rule. 
Using latest version: **0.4.0-beta**
```Cannot read property 'value' of undefined
TypeError
TypeError: Cannot read property 'value' of undefined
    at HTMLInputElement.handler (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:5659:21)
    at HTMLInputElement.dispatch (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:5429:27)
    at HTMLInputElement.elemData.handle (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:5233:28)
    at Object.trigger (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:8715:12)
    at HTMLInputElement.<anonymous> (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:8793:17)
    at Function.each (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:381:19)
    at jQuery.fn.init.each (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:203:17)
    at jQuery.fn.init.trigger (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:8792:15)
    at jQuery.fn.init.jQuery.fn.<computed> [as focus] (C:\Users\Gregory\AppData\Local\Programs\led\resources\app.asar\assets\js\jquery-3.5.1.js:10738:10)
    at HTMLDocument.<anonymous> (file:///C:/Users/Gregory/AppData/Local/Programs/led/resources/app.asar/assets/js/renderer.js:23127:21)```

**Comment by deepnight** [comment]
Thanks for reporting, I'll see that tomorrow :)

**Comment by deepnight** [comment]
Can your crash be reproduced easily? 🤔  If so, could please send me the faulty Json, if it's ok for you?

**Comment by videogreg93** [comment]
@deepnight Yeah, whenever I try to enter one of the values manually instead of a slider it happens. I can send you the json, do you also need the tilesets to open the project?

**Comment by deepnight** [comment]
No, the json should be enough 👍 
Thank you!

**Comment by videogreg93** [comment]
How should I send it to you? I cant send it here apparently and it's too big for pastebin.

**Comment by videogreg93** [comment]
[jsonCrash.zip](https://github.com/deepnight/led/files/5435976/jsonCrash.zip)

**Comment by deepnight** [comment]
Related to jQuery bug: https://github.com/jquery/jquery/issues/4417

**Comment by deepnight** [comment]
It will be fixed in 0.4.1 update.

**Comment by mgol** [comment]
I'd strongly advise not to depend on the jQuery Git version; breaking changes may happen there any moment. This version is mostly meant for testing.

The issue mentioned here is fixed in jQuery 3.6.0 so you can update to that instead.

**Comment by deepnight** [comment]
Perfect, thank you :)
